### DISCUSSION TITLE
Barrier "forgets" which system I'm on

### DISCUSSION THREAD

**Discussion by rtuite** [debauchee/barrier#1579] [Q&A]
I may have posted this before, but I can't seem to find it so I may not have.

Also, I can't deliberately recreate the issue, so I'm not sure what's causing it. I'd like to track that down though. It does happen independently of the client machine. I have not tried a new server yet.

What happens:
Occasionally when I transition from the bottom 3 screens (server, Win10) to the top 3 screens (client, now Win11 (was Win10 until today, replaced 2nd system)), I will lose the ability to control both. It is as though the server thinks it's inputting on the client, but the client is not receiving it. However, I can no longer control the server when this happens. I've tried remoting into the server using AnyDesk, in which case I am still unable to control the server from the remote connection. 

If I plug a keyboard/mouse into the client, I can operate it normally.

The only thing that fixes this is a reboot of the server. I can still do Ctrl+Alt+Del to get a security screen, and I can reboot from there. I can also do things like lock the machine, but the issue persists on unlock. I can also launch Task Manger, but I can't interact because the machine is not allowing input. This has happened to me twice in the last 3 days.

**Comment by myn** [comment]
I too have this problem.  3 Screens on the top running MacOS 12.2 and 3 screens on the bottom running Windows 10.  Happens randomly and been trying to reproduce but have yet to.

If I restart the client (top mac) it seems to fix it temporarily.
